### ISSUE TITLE
stuck on safe mode

### ISSUE THREAD

**Issue by wyup** [PixelExperience/android-issues#5943]
### Build Date

PixelExperience_Plus_dipper-13.0-20231122-1748-OFFICIAL.zip

### Device

dipper

### Version

thirteen_plus

### Describe the Bug

Safe mode is a black screen with 'Safe Mode' title on down left corner. Can't do anything.
When I press power and power menu appears, even if i short press Reboot or Power off ,the phone reboots again in Safe Mode. I can't make it boot normal mode again.

### Steps to Reproduce

From Power Menu in Android 13, long press 'Restart' and Reboot in Safe Mode appears. Phone Reboots to safe mode, but it is only a black screen with nothing to do. When I press power button again for power mode, I short press 'Reboot' or Power Off', but after that the phone boots to Safe Mode again, and can't get to normal boot again.

i have tried wiping dalvik-cache and cache from Recovery to no avail.
It looks the only way is to clear user data and reinstall.

### Expected Behavior

Boot normal mode after 'Reboot' from Safe Mode

### Possible Solution

Correct Safe Mode with functionality and Reboot to normal mode.

**Comment by pixelexperiencebot** [comment]
Issue created! You can close at any time by commenting ```/close```

**Comment by GiaSen** [comment]
I think there's an issue with your buttons, because safe mode only enables with volume down buttons if I recall correctly

try to press volume+ and other stuff to fix that

**Comment by wyup** [comment]
My volume buttons are ok. Power button yes, it acts up, but not all the time. I never came out of Safe Mode, which is just this title on a black screen.,
I entered Safe Mode like on this video, by long-pressing 'Reboot' from normal Power menu:
[youtube.com/watch?v=HK38IIrdFtc](youtube.com/watch?v=HK38IIrdFtc)
Tried rebooting into Bootloader/Fastboot, PixelExperience Recovery and I even installed OrangeFox Recovery. And fixed all partitions, cleaned Cache and Dalvik, but nothing. Only way out is factory reset.
